test/test_transforms.py

```diff
@@ -1997,5 +1757,245 @@ def test_random_order():
+
+    assert len(results) == 10
+    assert results == expected_output
+
+
+def test_five_crop():
+    to_pil_image = transforms.ToPILImage()
+    h = random.randint(5, 25)
+    w = random.randint(5, 25)
+    for single_dim in [True, False]:
+        crop_h = random.randint(1, h)
+        crop_w = random.randint(1, w)
+        if single_dim:
+            crop_h = min(crop_h, crop_w)
+            crop_w = crop_h
+            transform = transforms.FiveCrop(crop_h)
+        else:
+            transform = transforms.FiveCrop((crop_h, crop_w))
+
+        img = torch.FloatTensor(3, h, w).uniform_()
+
+        results = transform(to_pil_image(img))
+
+        assert len(results) == 5
+        for crop in results:
+            assert crop.size == (crop_w, crop_h)
+
+        to_pil_image = transforms.ToPILImage()
+        tl = to_pil_image(img[:, 0:crop_h, 0:crop_w])
+        tr = to_pil_image(img[:, 0:crop_h, w - crop_w:])
+        bl = to_pil_image(img[:, h - crop_h:, 0:crop_w])
+        br = to_pil_image(img[:, h - crop_h:, w - crop_w:])
+        center = transforms.CenterCrop((crop_h, crop_w))(to_pil_image(img))
+        expected_output = (tl, tr, bl, br, center)
+        assert results == expected_output
+
+
+@pytest.mark.parametrize('policy', transforms.AutoAugmentPolicy)
+@pytest.mark.parametrize('fill', [None, 85, (128, 128, 128)])
+def test_autoaugment(policy, fill):
+    random.seed(42)
+    img = Image.open(GRACE_HOPPER)
+    transform = transforms.AutoAugment(policy=policy, fill=fill)
+    for _ in range(100):
+        img = transform(img)
+        transform.__repr__()
+
+
+def test_center_crop():
+    height = random.randint(10, 32) * 2
+    width = random.randint(10, 32) * 2
+    oheight = random.randint(5, (height - 2) / 2) * 2
+    owidth = random.randint(5, (width - 2) / 2) * 2
+
+    img = torch.ones(3, height, width)
+    oh1 = (height - oheight) // 2
+    ow1 = (width - owidth) // 2
+    imgnarrow = img[:, oh1:oh1 + oheight, ow1:ow1 + owidth]
+    imgnarrow.fill_(0)
+    result = transforms.Compose([
+        transforms.ToPILImage(),
+        transforms.CenterCrop((oheight, owidth)),
+        transforms.ToTensor(),
+    ])(img)
+    assert result.sum() == 0
+    oheight += 1
+    owidth += 1
+    result = transforms.Compose([
+        transforms.ToPILImage(),
+        transforms.CenterCrop((oheight, owidth)),
+        transforms.ToTensor(),
+    ])(img)
+    sum1 = result.sum()
+    assert sum1 > 1
+    oheight += 1
+    owidth += 1
+    result = transforms.Compose([
+        transforms.ToPILImage(),
+        transforms.CenterCrop((oheight, owidth)),
+        transforms.ToTensor(),
+    ])(img)
+    sum2 = result.sum()
+    assert sum2 > 0
+    assert sum2 > sum1
+
+
+def test_center_crop_2():
+    """ Tests when center crop size is larger than image size, along any dimension"""
+    even_image_size = (random.randint(10, 32) * 2, random.randint(10, 32) * 2)
+    odd_image_size = (even_image_size[0] + 1, even_image_size[1] + 1)
```

Comment:
to parametrize over `input_images_sizes` and preserve the current logic, we can  do something like:

```py
@pytest.mark.parametrize('odd_size', (True, False))
def test_center_crop_2(odd_size, ...):
    input_image_size = (random.randint(10, 32) * 2, random.randint(10, 32) * 2)
    if odd_size:
        input_image_size = (input_image_size[0] + 1, input_image_size[1] + 1)
```